### PR TITLE
A little Progress

### PR DESCRIPTION
**Pull request by XRSHEERAN**
Finished general looking of WPF version LoadResult Form. Still working on the methods, which require a couple of other forms. Open this pull for reviewing and will update this pull from time to time. Thanks!

### CODE REVIEW

File: ProteoWPFSuite/loadResults.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+using System.Drawing;
+using System.Reflection;
+using System.IO;
+using Xceed.Wpf.Toolkit;
+namespace ProteoWPFSuite
+{
+    /// <summary>
+    /// Interaction logic for loadResults.xaml
+    /// </summary>
+    public partial class LoadResults : Window
```
Comment (acesnik) [inline ProteoWPFSuite/loadResults.xaml.cs]: Could you rename this file `LoadResults.xaml.cs`? The capitalization is off.

File: ProteoformSuiteGUI/ProteoformSweet.cs [hunk 2]
```diff
@@ -453,5 +453,10 @@ private void resultsToolStripMenuItem_Click(object sender, EventArgs e)
         {
 
         }
+
+        private void menuStrip1_ItemClicked(object sender, ToolStripItemClickedEventArgs e)
```
Comment (acesnik) [inline ProteoformSuiteGUI/ProteoformSweet.cs]: Could you remove this click event?

File: ProteoWPFSuite/loadResults.xaml [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                        <Grid x:Name="panel_step" Grid.Column="0">
+                            <Button x:Name="bt_stepthru" Margin="0 0 0 0" Content="Step Through Progress"/>
+                        </Grid>
+                        <Grid x:Name="panel_fullrun" Grid.Column="1">
+                            <Button x:Name="bt_fullrun" Content="Full Run" Margin="2 0 2 0"/>
+                        </Grid>
+                    </Grid>
+                </GroupBox>
+            </Grid>
+        </GroupBox>
```
Comment (acesnik) [inline ProteoWPFSuite/loadResults.xaml]: This is intense! Can't wait to see it :)

File: ProteoformSuiteInternal/ProteoformSuiteInternal.csproj [hunk 4]
```diff
@@ -65,7 +65,7 @@
       <HintPath>..\packages\FastMember.Signed.1.3.0\lib\net45\FastMember.Signed.dll</HintPath>
     </Reference>
     <Reference Include="FlashLFQ, Version=1.0.0.0, Culture=neutral, processorArchitecture=AMD64">
-      <HintPath>..\packages\mzLib.1.0.298\lib\net45\FlashLFQ.dll</HintPath>
+      <HintPath>..\packages\FlashLFQ.0.1.100\lib\net461\FlashLFQ.dll</HintPath>
```
Comment (acesnik) [inline ProteoformSuiteInternal/ProteoformSuiteInternal.csproj]: Is this change in line with what is coming down the pipe from mzLib?

File: ProteoformSuiteGUI/LoadResults.Designer.cs [hunk 5]
```diff
@@ -36,22 +36,22 @@ private void InitializeComponent()
...
             this.cb_calibrate_td_files = new System.Windows.Forms.CheckBox();
             this.cb_calibrate_raw_files = new System.Windows.Forms.CheckBox();
-            this.param_splitcontainer = new System.Windows.Forms.SplitContainer();
-            this.label_minRT = new System.Windows.Forms.Label();
             this.label_maxRT = new System.Windows.Forms.Label();
-            this.label_mincharge = new System.Windows.Forms.Label();
-            this.label_maxcharge = new System.Windows.Forms.Label();
-            this.nud_mincharge = new System.Windows.Forms.NumericUpDown();
-            this.nud_maxcharge = new System.Windows.Forms.NumericUpDown();
+            this.label_minRT = new System.Windows.Forms.Label();
```
Comment (acesnik) [inline ProteoformSuiteGUI/LoadResults.Designer.cs]: Are these changes intentional, or are they reverting the changes made to the LoadResults GUI from earlier this year?

File: ProteoformSuiteInternal/ProteoformSuiteInternal.csproj (hunk 4 above)
Comment (XRSHEERAN) [reply ProteoformSuiteInternal/ProteoformSuiteInternal.csproj]: Yes, I think I have the wrong version for mzlib. Maybe just neglect it, thanks!

File: ProteoformSuiteGUI/LoadResults.Designer.cs (hunk 5 above)
Comment (XRSHEERAN) [reply ProteoformSuiteGUI/LoadResults.Designer.cs]: Sorry I didn't intend to change any part of the ProteoforSuiteGUI. I'll go clean up the unwanted changes tomorrow!

File: ProteoformSuiteInternal/ProteoformSuiteInternal.csproj (hunk 4 above)
Comment (acesnik) [reply ProteoformSuiteInternal/ProteoformSuiteInternal.csproj]: No worries. I changed this and a bunch of other package listings. I'll commit that after merging this.

File: ProteoformSuiteGUI/LoadResults.Designer.cs (hunk 5 above)
Comment (acesnik) [reply ProteoformSuiteGUI/LoadResults.Designer.cs]: Got it! Thanks for the note. I reverted these changes to the Windows Forms GUI. I'll commit those after merging this.
